bootstrap-orbit.sh

```diff
@@ -98,7 +128,8 @@ else
   echo "Found conan. Checking version..."
   CONAN_VERSION="$(conan --version | cut -d' ' -f3)"
   
-  if [[ "$CONAN_VERSION" < "$CONAN_VERSION_REQUIRED" ]]; then
+  check_conan_version_sufficient "$CONAN_VERSION" "$CONAN_VERSION_REQUIRED"
+  if [ $? -ne 0 ]; then
```

Comment:
This can be written as `if ! check_conan_version_sufficient "$CONAN_VERSION" "$CONAN_VERSION_REQUIRED"; then`

Reply:
done